neo/Persistence/Snapshot.cs

```diff
@@ -35,6 +35,8 @@ public abstract class Snapshot : IDisposable, IPersistence, IScriptTable
         public UInt256 CurrentBlockHash => BlockHashIndex.Get().Hash;
         public UInt256 CurrentHeaderHash => HeaderHashIndex.Get().Hash;
 
+        private const uint NoBonusHeight = 200_000;
```

Comment:
Now we decide to chage it to `8_000_000`

Reply:
Have changed

Reply:
> Have changed

Format